woocommerce/woocommerce-accommodation-bookings
# Fix QIT linting errors

### Changes proposed in this Pull Request:

Minor code updates across several files to fix linting errors found by QIT CLI. The biggest change is to how some $_POST inputs are handled and sanitized.

### How to test the changes in this Pull Request:

Basically there should be no visual changes from this. Check to make sure none of the admin UI for Accommodation Bookings shows any escaped HTML tags or other glitches and that the form fields work correctly.

1. Make sure WooCommerce Bookings is installed and activated along with this plugin
2. Add a new product
3. Under Product Data change the product type to Accommodation product
4. Check the boxes for Has Persons and Has Resources
5. Fill in some data for the product fields relevant to accommodations
6. On the Persons tab, check the box for Enable Person Types
7. Add two or three person types and fill in their data
8. On the Resources tab, add two or three resources and fill in their data
9. Publish the new product
10. When the screen has refreshed, make sure all the data you filled in is still present in the form fields
11. Change some of the person and resource data and update the product
12. When the screen has refreshed, make sure the changes you made have persisted

### Changelog entry

> Dev - Fix linting errors found by the Quality Insights Toolkit.
